src/components/molecules/GameCard/GameCard.js

```diff
@@ -13,21 +14,61 @@ export function GameCard({
     gameTitle = 'Omega Long Game Title',
     releaseDate = '12/22/22',
 }) {
+    const { ratingStyle, sideBarStyle } = useMemo(() => {
+        if (typeof rating !== 'number')
+            return {
+                ratingStyle: styles.noRating,
+                sideBarStyle: styles.sideBarNoRating,
+            }
+
+        switch (true) {
+            case rating >= 75:
```

Comment:
I found a fun bug in the colors spit out from this guy. Since we're rounding the rating to the nearest whole number, the UI can round a number like 74.89 to 75, but when determining these colors a 75 will still be yellow, even though we're checking `rating >= 75`.

The solution: We also need to round rating up here in this function. Probably worth doing before the switch statement so we only have to do it once.

Reference the image below.
![image](https://user-images.githubusercontent.com/33842042/212788683-a7780478-9053-4f6c-a811-f63c124d21b5.png)